templates/repo/issue/sidebar/label_list.tmpl

```diff
@@ -43,7 +43,7 @@
 		</div>
 	</div>
 
-	<div class="ui list labels-list tw-my-2 tw-flex tw-gap-2">
+	<div class="ui list labels-list tw-my-2 tw-flex tw-flex-wrap">
```

Comment:
It seems to be an abuse by mixing "ui list" and "labels-list"

Will take a look

Reply:
Yeah let's remove `.ui.list`

Reply:
It can't just remove that. See repo-issue-sidebar.md for more details.

Away from keyboard and will take a look in hours.